ProcessOut/Classes/AuthorizationRequest.swift

```diff
@@ -7,14 +7,17 @@
 
 class AuthorizationRequest: Codable {
     var source: String = ""
+    var incremental: Bool = false
```

Comment:
Do these properties need to be mutable?

Reply:
As far as I'm aware there isn't any specific reason that these properties to be set as mutable, but the accepted practice for classes and structs that conform to `Codable` appears to be to use `var` as seen here. This is seen in the Swift docs:

https://developer.apple.com/documentation/foundation/archives_and_serialization/encoding_and_decoding_custom_types

It's possible I've misunderstood that article, so I would be interested to hear if what I've said is consistent with your understanding of Swift best practices.